crates/testing/tests/standalone_integration_test.rs

```diff
@@ -43,39 +43,34 @@ async fn read_logs(module: &ModuleHandle) -> Vec<String> {
 fn test_calling_a_reducer_in_module(module_name: &'static str) {
     init();
 
-    CompiledModule::compile(module_name, CompilationMode::Debug).with_module_async(
```

Comment:
clippy really wanted to reformat this. It helps to review with whitespace hidden, but it's still not great.